subfunctions/g_CaBBI_observation

```diff
@@ -0,0 +1,19 @@
+-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Here again the derivative should be super easy to derive analytically.
The options.checkGrads can help you check that the computations are correct.